addons/website/views/snippets/s_list_group.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_list_group" name="List Group">
+    <div class="s_list_group my-3" style="--s_list_group-icon-bg: color-mix(in srgb, currentColor 10%, transparent);">
```

Comment:
The style you set here is in fact what the `fade-current-color` CSS function does, so let's use it directly :+1: :arrow_down: 
- Remove the style here
- In the SCSS file, do this:
  ```scss
  .s_list_group {
    --s_list_group-icon-color: var(--o-cc1-btn-primary);
    --s_list_group-icon-bg: #{fade-currentColor(10%)};  
    ...
  ```
=> This will allow to always have a faded version of the chosen icon color by default, without having to select it manually in the background color option, which is nice because the user would have a hard time trying to match them otherwise :slightly_smiling_face: And the user can still choose the color he wants for the background, since it will override the variable :wink: